pkg/authorization/registry/rolebinding/rest.go

```diff
@@ -73,6 +73,9 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	if err := r.validateReferentialIntegrity(ctx, roleBinding); err != nil {
 		return nil, err
 	}
+	if err := r.confirmRoleBindingNameUnique(ctx, roleBinding.Name); err != nil {
+		return nil, err
+	}
```

Comment:
yay post-merge comments! for posterity, seems like we should do things in this order:
1. look up the policy binding
2. check for unique rolebinding name in the binding we retrieved
3. add the rolebinding to the policy binding
4. update the policy binding

That guarantees no other changes happened to the policy binding in between checking for uniqueness and doing the create. It does mean a "create rolebinding" operation would fail with a conflict if any other change to the policy binding occurred in that interval, which is unfortunate and hard to understand as a caller.